### PR TITLE
Add utils for common class tests (related to issue #62)

### PR DESCRIPTION
**Pull request by Gernene**
Add helper classes for common class tests. Splitting #62  into smaller PRs that are more manageable.

These files should not affect any existing classes or app functionality.

### CODE REVIEW

**Review by lkldev — APPROVED** [review]
Great Utils! thanks Gernene :)

File: src/test/java/seedu/address/testutil/TAssistBuilder.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package seedu.address.testutil;
+
+import seedu.address.model.TAssist;
+import seedu.address.model.student.Student;
+
+/**
+ * A utility class to help with building Addressbook objects.
+ * Example usage: <br>
+ *     {@code AddressBook ab = new AddressBookBuilder().withPerson("John", "Doe").build();}
```
Comment (jxt00) [inline src/test/java/seedu/address/testutil/TAssistBuilder.java:9]: Should these be adapted to `TAssist` instead?

File: src/test/java/seedu/address/testutil/TAssistBuilder.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    public TAssistBuilder() {
+        tassist = new TAssist();
+    }
+
+    public TAssistBuilder(TAssist tassist) {
+        this.tassist = tassist;
+    }
+
+    /**
+     * Adds a new {@code Person} to the {@code AddressBook} that we are building.
```
Comment (jxt00) [inline src/test/java/seedu/address/testutil/TAssistBuilder.java:24]: Should `Person` be `Student` instead?

File: src/test/java/seedu/address/testutil/TypicalClassGroups.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import seedu.address.model.classgroup.ClassGroup;
+
+/**
+ * A utility class containing a list of {@code Person} objects to be used in tests.
+ */
+public class TypicalClassGroups {
+    public static final ClassGroup CS2101G09 = new ClassGroupBuilder().withClassGroupId("A01")
+            .withClassGroupType("SECTIONAL").withModule(TypicalModules.CS2101).build();
+
+    public static final String KEYWORD_MATCHING_MEIER = "Meier"; // A keyword that matches MEIER
```
Comment (jxt00) [inline src/test/java/seedu/address/testutil/TypicalClassGroups.java:16]: Should `Person` be `ClassGroup` instead? I don't think line 16 is needed for `ClassGroup`.

File: src/test/java/seedu/address/testutil/TypicalModules.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
+package seedu.address.testutil;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.model.tamodule.TaModule;
+
+/**
+ * A utility class containing a list of {@code Person} objects to be used in tests.
```
Comment (jxt00) [inline src/test/java/seedu/address/testutil/TypicalModules.java:10]: Should `Person` be `TaModule` instead?

File: src/test/java/seedu/address/testutil/TypicalStudents.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
+package seedu.address.testutil;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.model.student.Student;
+
+/**
+ * A utility class containing a list of {@code Person} objects to be used in tests.
```
Comment (jxt00) [inline src/test/java/seedu/address/testutil/TypicalStudents.java:10]: Should `Person` be `Student` instead?

File: src/test/java/seedu/address/testutil/TypicalTAssist.java [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+import seedu.address.model.TAssist;
+import seedu.address.model.student.Student;
+
+public class TypicalTAssist {
+
+    private TypicalTAssist() {}
+
+    /**
+     * Returns an {@code AddressBook} with all the typical persons.
```
Comment (jxt00) [inline src/test/java/seedu/address/testutil/TypicalTAssist.java:11]: Should these be adapted to `TAssist` instead?

**Review by jxt00 — COMMENTED** [review]
Thanks for creating the builders!

**Review by jxt00 — APPROVED** [review]
Will merge first, headers can be fixed at a later date. 👍

**Pull request merged**